src/SkillsForm.jsx

```diff
@@ -51,23 +45,41 @@ class SkillForm extends React.Component {
 
   render() {
     const onSkillNameChange = ev => this.onChange("skill_name", ev.target.value);
-    const onSkillTypeChange = ev => this.onChange("skill_type", ev.value);
+    const onSkillTypeChange = ev => this.onChange("skill_type", ev.target.value);
+    const options = skillTypes.map(type =>
+       (<option key={type.value} value={type.value}>{type.label}</option>)
+       );
     return (
-      <form onSubmit={ev => this.onSubmit(ev)}>
-        <div>
-          Name: <input name="skill_name" 
-                       type="text" 
-                       value={this.state.skill_name} 
-                       onChange={onSkillNameChange} />
-        </div>
-        <div>
-          Skill Type: <Select name="skill_type" 
-                              value={this.state.skill_type} 
-                              onChange={onSkillTypeChange} 
-                              options={skillTypes} />
-        </div>
-        <button type="submit">Save</button>
-      </form>
+      <div>
+        <h1>Add Skill</h1>
+        <Form horizontal onSubmit={this.onSubmit}>
+          <FormGroup controlId='skillName'>
+            <Col componentClass={ControlLabel} sm={2}>
+              Name:
+            </Col>
+            <Col sm={10}>
+              <FormControl name='skillName' onChange={onSkillNameChange}/>
+            </Col>
+          </FormGroup>
+          <FormGroup controlId='skillType'>
+            <Col componentClass={ControlLabel} sm={2}>
+              Type:
+            </Col>
+            <Col sm={10}>
+              <FormControl componentClass="select" onChange={onSkillTypeChange}>
```

Comment:
Nice job removing a dependency for this component! 👍 I imagine we could do this for other components relying on `Select` elements...